geomet/wkb.py

```diff
@@ -291,8 +291,11 @@ def loads(string):
     elif endianness == LITTLE_ENDIAN:
         big_endian = False
     else:
-        raise ValueError("Invalid endian byte: '0x%s'. Expected 0x00 or 0x01"
-                         % binascii.hexlify(endianness.encode()).decode())
+        raise ValueError(
```

Comment:
I prefer the package stay consistent, and use fstrings over .format.

Reply:
Okay, I'm not attached to this change.